4001-5000/LIT4859Litany.xml

```diff
@@ -1,12 +1,12 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT4859Litany" xml:lang="en" type="work">
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">ሊጦን፡ ዘሠሉስ፡</title>
```

Comment:
It seems to be attested in several manuscripts as for Wednesday (not Tuesday...). It the same text is really attested for different days of the week, it would probably good to indicate this in the abstract. It is possible that the ID has been used in cases where the catalogues only said "liton for Tuesday" - I did not check all instances now.

Reply:
The use for various days is possible, in theory, but will require a small research. I have confirmed ehud with two contemporary printed sources.

Reply:
In ESam007 and FSUor15 this ID is used for rabuʿ, also in LIT4860Litanies. It's possible that there is a lot of confusion in the records.